.fernignore

```diff
@@ -1 +1,3 @@
 # Specify files that shouldn't be modified by Fern
+
+tests/custom/test_client.py
```

Comment:
@carenthomas i would also add `.github/workflows/ci.yml` to the fernignore as well

Reply:
added!